### PR TITLE
Intmdb 634 federatesettings orgrolemapping Required field and Doc update

### PR DESCRIPTION
**Pull request by govardhanpagidi**
## Description

Required field change in the Schema and ReadMe update

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code
- [x] For CFN Resources: I have released by changes in the private registry and proved by change works in Atlas

### CODE REVIEW

File: cfn-resources/federated-settings-org-role-mapping/test/inputs_1_create.template.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "Profile": "default",
+  "Profile": "federation",
```
Comment (andreaangiolillo) [inline cfn-resources/federated-settings-org-role-mapping/test/inputs_1_create.template.json:2]:
```suggestion
  "Profile": "default",
```

**Review by andreaangiolillo — COMMENTED** [review]
one small comment

**Comment by govardhanpagidi** [comment]
@andreaangiolillo  Currently the give ATLAS Org doesn't have FederatedSettings enabled, hence I created another profile with name 'federation'. If the above situation occurs we have to change the profile name and do the PUBLISHING from MASTER branch.

Anyhow this is not a major concern, just to remind why the change is pushed. The last PUBLISH happened from this feature branch with 'federation' profile name.

**Review by andreaangiolillo — APPROVED** [review]
LGTM. Thanks for the clarification.

**Pull request merged**